DEVGUIDE.md

```diff
@@ -0,0 +1,771 @@
+
+```
+    def dictionary_with_result_file(self):
+        path = Path.cwd() / 'resources/requests.csv'
+        dictionary = dict()
+        dictionary['result_file'] = path
+        return dictionary
+
+    def setUp(self):
+        config = {'handler': 'LocustHandler', 'keyword': 'run_locust', 'tags': 'LOCUST'}
+        self.handler = LocustHandler(config)
+        dictionary = self.dictionary_with_result_file()
+        dictionary['failure_percentage'] = None
+        self.test_suite = self.handler.parse_results(dictionary)
+```
+
+and run tests again. Still two test cases fail. This is because the `_get_treshold_failure_percentage` has an argument now instead of reading the value from the config. Let's update the failing test cases: 
+
+```
+    def test_failure_percentage_is_ten_by_default(self):
+        failure_percentage = self.handler._get_treshold_failure_percentage(None)
+        self.assertEqual(failure_percentage, 10)
+
+    def test_failure_percentage_max_amount_is_one_hundred(self):
+        failure_percentage = self.handler._get_treshold_failure_percentage(101)
+        self.assertEqual(failure_percentage, 100)
+```
+
+Now all tests should pass. Let's add two more test cases to see that the `failure_precentage` is set correctly from the parameter or config file.
+
+```
+    def test_parse_results_takes_failure_percentage_from_parameter_prior_to_config(self):
+        config = {'handler': 'LocustHandler', 'keyword': 'run_locust', 'tags': 'LOCUST', 'failure_percentage': '70'}
+        self.handler = LocustHandler(config)
+        dictionary = self.dictionary_with_result_file()
+        dictionary['failure_percentage'] = 75
+        test_suite = self.handler.parse_results(dictionary)
+        self.assertEqual(test_suite['name'], 'Locust test suite, failure percentage 75')
+
+    def test_parse_results_takes_failure_percentage_correctly_from_config(self):
+        config = {'handler': 'LocustHandler', 'keyword': 'run_locust', 'tags': 'LOCUST', 'failure_percentage': '70'}
+        self.handler = LocustHandler(config)
+        dictionary = self.dictionary_with_result_file()
+        dictionary['failure_percentage'] = None
+        test_suite = self.handler.parse_results(dictionary)
+        self.assertEqual(test_suite['name'], 'Locust test suite, failure percentage 70')
+```
+
+All 8 tests should pass. Now we have completed an LocustHandler with unit tests which test the most important functionalities.
+
+
+
+## How to package your project
+
+Let's package our project in the same virtual environment . Add necessary files defined in this [tutorial](https://packaging.python.org/) to `locustenv` folder and install `setuptools` and `wheel` to your virtualenv with `pip install`. Add following line yo your `setup.py` file in the `setuptools.setup() object`:
```

Comment:
`setuptools` and `wheel` are part of Python standard library ie. installed by default. Please remove sentence about installing them.

Reply:
`wheel` needs to be installed